Battery tile crashes everytime and shows red exclamation mark

https://drive.google.com/file/d/0B0NJb0mKnh5QT1d4MF9MS2VnLU0/view?usp=drivesdk
Here is the logcat, system ui crashes on battery tile click in cm12.1 and shows an exclamation mark on battery tile

The SystemUI doesn't have the battery permission for some reason, thus the crash.

```
08-31 20:45:46.068 E/AndroidRuntime(717): Caused by: java.lang.SecurityException: uid 10021 does not have android.permission.BATTERY_STATS.
08-31 20:45:46.068 E/AndroidRuntime(717): 	at android.os.Parcel.readException(Parcel.java:1546)
08-31 20:45:46.068 E/AndroidRuntime(717): 	at android.os.Parcel.readException(Parcel.java:1499)
08-31 20:45:46.068 E/AndroidRuntime(717): 	at com.android.internal.app.IBatteryStats$Stub$Proxy.getStatisticsStream(IBatteryStats.java:1063)
08-31 20:45:46.068 E/AndroidRuntime(717): 	at com.android.internal.os.BatteryStatsHelper.getStats(BatteryStatsHelper.java:1034)
08-31 20:45:46.068 E/AndroidRuntime(717): 	at com.android.internal.os.BatteryStatsHelper.loadStats(BatteryStatsHelper.java:1005)
08-31 20:45:46.068 E/AndroidRuntime(717): 	at com.android.internal.os.BatteryStatsHelper.getStats(BatteryStatsHelper.java:237)
08-31 20:45:46.068 E/AndroidRuntime(717): 	at tk.wasdennnoch.androidn_ify.extracted.settingslib.BatteryInfo$2.doInBackground(BatteryInfo.java:112)
```

--- a/app/src/main/java/tk/wasdennnoch/androidn_ify/systemui/SystemUIHooks.java
+++ b/app/src/main/java/tk/wasdennnoch/androidn_ify/systemui/SystemUIHooks.java
@@ -97,6 +97,13 @@ public class SystemUIHooks {
                 }, 2000);
             }
         });
+        
+        XposedHelpers.findAndHookMethod("android.app.ContextImpl", classLoader, "enforceCallingPermission", String.class, String.class, new XC_MethodHook() {
+            @Override
+            protected void beforeHookedMethod(MethodHookParam param) throws Throwable {
+                if (param.args[0].equals("android.permission.BATTERY_STATS")) param.setResult(null);
+            }
+        });
 
     }
 

